app/javascript/components/cloud-volume-form/cloud-volume-form.schema.js

```diff
@@ -158,7 +159,7 @@ const createSchema = (fields, edit, ems, loadSchema, emptySchema) => {
             isRequired: true,
             validate: [
               { type: validatorTypes.REQUIRED },
-              async (value) => validateName("storage_services", value)
+              async (value) => validateName("storage_services", value),
```

Comment:
avoid space after async and use single quotes for parameters
`async(value) => validateName('storage_services', value),`